README.md

```diff
@@ -1,2 +1,121 @@
+
+1. Install [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+2. Open a pull request on this repository in which you duplicate `config/prod/example-project.yaml` as `config/prod/<stack_name>.yaml` and follow the numbered steps listed in the file. Note that some steps are required whereas others are optional.
+
+3. Once the pull request is approved and merged, open a terminal and perform the following steps:
+
+   1. [Copy](https://d-906769aa66.awsapps.com/start#/) the temporary credentials for your Developer `sandbox` role, available under:
+      ```
+      AWS Accounts > org-sagebase-sandbox > Developer > Command line or programmatic access > Option 1
+      ```
+      Run the copied `export` commands in the terminal, which should look like this:
+      ```
+      export AWS_ACCESS_KEY_ID="..."
+      export AWS_SECRET_ACCESS_KEY="..."
+      export AWS_SESSION_TOKEN="..."
+      ```
+
+      <!-- TODO: Update link with our production instance of Tower -->
+   2. [Create](https://tower-dev.sagebionetworks.org/tokens) a new token in Nextflow Tower called `<stack_name>`, copy the token (which is only displayed once), and run the following command, updating `<token>` with the copied value:
+      ```
+      export NXF_TOWER_TOKEN="<token>"
+      ```
+
+   3. [Create](https://www.synapse.org/#!PersonalAccessTokens:) a new personal access token in Synapse called `<stack_name>` with all scopes enabled, copy the token (which is only displayed once), and run the following command, updating `<token>` with the copied value:
+      ```
+      export SYNAPSE_TOKEN="<token>"
+      ```
+
+   4. Run the following Docker command to configure your project in Nextflow Tower, updating `<stack_name>` with the stack name:
+      ```
+      docker run -e STACK_NAME="<stack_name>" -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e NXF_TOWER_TOKEN -e SYNAPSE_TOKEN -v "$HOME/.aws:/root/.aws" sagebionetworks/setup-tower-project > project-config.json
+      ```
+
+4. The above Docker command performs the following tasks:
+
+   - In your AWS CLI configuration (`~/.aws/config`), it configures two profiles if they don't already exist:
+     - `sandbox` for using the `Developer` role from the `sandbox` AWS account
+     - `tower` for using the `TowerViewer` role from the `nextflow-prod` AWS account
+
+     You can login using these profiles with the following commands
+     ```
+      aws --profile sandbox sso login
+      aws --profile tower sso login
+     ```
+
+   - In Nextflow Tower, the credentials for the Forge service user were created under the name `<stack_name>`.
+
+   - In Nextflow Tower, a compute environment was created using the above credentials under the name `<stack_name> (default)`. If you need to tweak the compute environment, we recommend that you clone the default one and make adjustments as necessary.
+
```

Comment:
Maybe a final step to try an easy nextflow tower command?

Reply:
I agree that these docs end abruptly. I'll be extended them once I work through these steps for a real project (_e.g._ imCORE). I'll update the setup script to launch a run using [nextflow-io/hello](https://github.com/nextflow-io/hello) and provide a URL for them to visit to see it complete.